setup.py

```diff
@@ -69,7 +69,7 @@ def get_version():
     author_email='zimbler@gmail.com',
     description='Manticoresearch database backend for Django',
     install_requires=[
-        'Django>=3.1,<4.2',
```

Comment:
That is strange change. Why `2.2` appeared?

Oldest confirmed version in `tox.ini`  is `3.1`

Reply:
Yes, return.